website/docs/d/nsxt_edgegateway_bgp_neighbor.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_bgp_neighbor"
+sidebar_current: "docs-vcd-resource-nsxt-edgegateway-bgp-neighbor"
+description: |-
+  Provides a data source to read NSX-T Edge Gateway BGP Neighbors and their configuration.
+---
+
+# vcd\_nsxt\_edgegateway\_bgp\_neighbor
+
+Supported in provider *v3.7+* and VCD 10.2+ with NSX-T
+
+Provides a data source to read NSX-T Edge Gateway BGP Neighbors and their configuration.
+
+## Example Usage
+
+```hcl
+data "vcd_nsxt_edgegateway" "existing" {
+  org = "my-org"
+  vdc = "nsxt-vdc"
+
+  name = "nsxt-gw"
+}
+
+data "vcd_nsxt_edgegateway_bgp_neighbor" "first" {
+  org = "my-org"
+  vdc = "nsxt-vdc"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+
+  ip_address = "192.168.102.45"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to which the edge gateway belongs. Optional if defined at provider level.
+* `edge_gateway_id` - (Required) An ID of NSX-T Edge Gateway. Can be lookup up using
```

Comment:
```suggestion
* `edge_gateway_id` - (Required) An ID of NSX-T Edge Gateway. Can be looked up up using
```

Reply:
Thanks. Fixed here and in 5 more places.